user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_DiscoveryAndProvisioning.md

```diff
@@ -0,0 +1,160 @@
+---
+uid: IDP_Tutorial_DiscoveryAndProvisioning
+keywords: idp tutorial, idp kata
+---
+
+# Getting started with IDP and onboarding new equipment
+
+This tutorial will show you how to install IDP and onboard new equiment.
+
+Expected duration: 20 minutes.
+
+<!-- TODO: Uncomment and fill in with correct info when Kata is released.
+> [!TIP]
+> See also: [Kata #xx: IDP Introduction](https://community.dataminer.services/courses/kata-xx/)
+-->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.0-CU0 or higher that is
+
+  - [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+  - running on a Microsoft Windows server.
+
+> [!IMPORTANT]
+> If you are using a DataMiner version lower than 10.4.0 or 10.4.3, install the [IDP Migration](https://community.dataminer.services/download/idp-migration/) package first.
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
+
+- [Step 1: Deploy IDP from the Catalog](#step-1-deploy-idp-from-the-catalog)
+- [Step 2: Configure discovery profile](#step-2-configure-discovery-profile)
+- [Step 3: Configure scan range](#step-3-configure-scan-range)
+- [Step 4: Run the discovery](#step-4-run-the-discovery)
+- [Step 5: Provision the discovered element(s)](#step-5-provision-the-discovered-elements)
+
+## Step 1: Deploy IDP from the Catalog
+
+1. Go to [catalog.dataminer.services](https://catalog.dataminer.services).
+1. Search for the [IDP package](https://catalog.dataminer.services/details/package/3163).
+1. Deploy the package on your DataMiner system.
+1. Go to your DataMiner system, you'll notice that IDP will be automatically installed and set up for you.
+
+## Step 2: Configure discovery profile
+
+Discovery profiles define which information and how it should be retrieved from the device. Some devices require authentication, which needs to be filled in upfront.
```

Comment:
The first sentence is strange...  something like this maybe ? "Discovery profiles define which information should be retrieved from the device."

Reply:
Changed, I probably changed my mind midsentence.